mkt/reviewers/tests/test_views.py

```diff
@@ -582,8 +583,10 @@ def test_queue_count(self):
 
     def test_homescreen_count(self):
         Tag(tag_text='homescreen').save_tag(self.apps[1])
+        self.apps[1].save()
         if self.uses_es():
-            self.reindex(Webapp)
+            WebappIndexer.unindex(self.apps[1].id)
```

Comment:
Is this necessary just because you changed the app to homescreens after indexation ? I assume it only happens in tests ?